tests/unit/RequestBuilder/RecommendationRequestBuilderTest.php

```diff
@@ -147,4 +156,12 @@ public function shouldFailOnIncorrectSequenceOfUsersWhenMerging(): void
         $builder->setInteraction($interactionCommand);
         $builder->build();
     }
+
+    public function correctSequenceOfUsersProvider()
```

Comment:
- please add return typehint
- also add phpdoc to help phpstorm & phpstan recognize internal type of array contents
- naming convence for provider method name is "provideFooBar", eg. to not conincidentally name them like "testUsersProvider", which would cause PHpUnit to recognize them as tests and do unexpected things

So this should be:
```php
     /**
     * @return array[]
     */
    public function providecorrectSequenceOfUsers(): array
```

Reply:
done